Source/Urho3D/UI/DropDownList.cpp

```diff
@@ -314,16 +330,18 @@ bool DropDownList::FilterPopupImplicitAttributes(XMLElement& dest) const
     return true;
 }
 
-void DropDownList::HandleItemClicked(StringHash eventType, VariantMap& eventData)
+void DropDownList::HandleItemClicked(StringHash /*eventType*/, VariantMap& /*eventData*/)

```

Comment:
Macro for unused arguments should be preferred. Function signature refactoring feature of IDEs is broken otherwise so it is a bad practice.

Reply:
[offtopic]
I'm ashamed to admit that I do such things too, sometimes.
When I write empty no-body function, it's really tempting to just comment out parameters.

Reply:
I adopted this method from Urho's code base. We could switch to using a macro and replace all `'/\*\w+\*/,'` - if we decide to do so - later.

Reply:
Maybe we could put the [lambda wrap](https://github.com/urho3d/Urho3D/pull/2593) on the menu as a generalized ignore function?